json-spec/examples/landsat8-sample.json

```diff
@@ -35,92 +35,195 @@
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B4.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["4"]
     },
     "B5": {
-      "href": "LC81530252014153LGN00_B5.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-5.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B5.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["5"]
     },
     "B6": {
-      "href": "LC81530252014153LGN00_B6.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-6.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B6.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["6"]
     },
     "B7": {
-      "href": "LC81530252014153LGN00_B7.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-7.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B7.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["7"]
     },
     "B8": {
-      "href": "LC81530252014153LGN00_B8.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-8.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B8.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["8"]
     },
     "B9": {
-      "href": "LC81530252014153LGN00_B9.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-9.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B9.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["9"]
     },
     "B10": {
-      "href": "LC81530252014153LGN00_B10.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-10.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B10.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["10"]
     },
     "B11": {
-      "href": "LC81530252014153LGN00_B11.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-11.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B11.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["11"]
     },
-    "metadata": {
-      "href": "LC81530252014153LGN00_MTL.txt",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product_metadata.json"
+
+    "eo:bands": {
+      "1": {
+        "common_name": "coastal",
+        "gsd": 30.0,
+        "accuracy": null,
+        "wavelength": 0.44,
+        "fwhm": 0.02
```

Comment:
I prefer the more explicit "center_wavelength" and "full_width_half_maximum"

Reply:
I agree with that - people who aren't super knowledgeable should be able to look at the names and get understanding of what it is.

Reply:
"center_wavelength" is barely tolerable with regard to length, "full_width_half_maximum" hardly, however I very much appreciate the desire to be explicit with names and, given that these are not going to be commonly used fields by most users, I will capitulate on this point.